backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/banneralert/BPKBannerAlert.kt

```diff
@@ -0,0 +1,59 @@
+/*
+* Backpack for Android - Skyscanner's Design System
+*
+* Copyright 2018 Skyscanner Ltd
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package net.skyscanner.backpack.compose.banneralert
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.banneralert.internal.BannerAlert
+import net.skyscanner.backpack.compose.icon.BpkIcon
+
+enum class BPKBannerAlertType {
+    Info,
+    Success,
+    Warning,
+    Error,
+}
+
+enum class BPKBannerAlertStyle {
+    Default,
+    OnContrast,
+}
+
+data class CustomIcon(
+    val icon: BpkIcon?,
```

Comment:
Why is icon field nullable? Could there be a case in which the icon doesn't exist but it has an accessiblityLabel?

Reply:
Ha, good point. Will update.